lib/jekyll-compose/file_creator.rb

```diff
@@ -0,0 +1,33 @@
+class Jekyll::Compose::FileCreator
+  attr_reader :file, :force
+  def initialize(fileInfo, force=false)
```

Comment:
Please place a space between parameters with defaults and the default value: `force = false`